### ISSUE TITLE
Proposal: separate journal storage options for system and user units

### ISSUE THREAD

**Issue by dsseng** [systemd/systemd#28933]
### Component

systemd-journald

### Is your feature request related to a problem? Please describe

To avoid excess drive write I use volatile journals. However, if the system sometimes hangs with no display output, there's no way to obtain logs.

### Describe the solution you'd like

It would be nice if I could make journald save&archive kernel logs, but store user units' logs in a volatile storage.

### Describe alternatives you've considered

Using different software for capturing and saving kernel logs as I'm particularly interested in those in case of issues.

### The systemd version you checked that didn't have the feature you are asking for

253.7-1.fc38

**Comment by russdill** [comment]
Yes please. Having the ability to store different groups of logs in different places with different quotas would be enormously helpful. I tried using --namespace= for this, and it *almost* does exactly what I want but it has the unfortunate side effect of putting the units into mount namespaces which is definitely not what I want.
